src/System.Net.Security/src/System/Net/SslStreamPal.Unix.cs

```diff
@@ -75,19 +75,9 @@ public static void QueryContextStreamSizes(SafeDeleteContext securityContext, ou
             streamSizes = new StreamSizes();
         }
 
-        public static int QueryContextConnectionInfo(SafeDeleteContext securityContext, out SslConnectionInfo connectionInfo)
+        public static void QueryContextConnectionInfo(SafeDeleteContext securityContext, out SslConnectionInfo connectionInfo)
```

Comment:
That's why I like PAL interfaces vs PAL static implicit call conventions, personally.